OpenRA.Game/Game.cs

```diff
@@ -855,6 +855,14 @@ static RunStatus Run()
 			return state;
 		}
 
+		public static void ExitIfNotRunningGame()
+		{
+			if (server == null || !server.IsStartedOrWaiting())
```

Comment:
This won't work in multiplayer games. You may be able to check something inside `OrderManager` instead.